apps/mochi-web/components/Profile/PaymeSection.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  TabContent,
+  TabList,
+  TabTrigger,
+  Tabs,
+  Typography,
+} from '@mochi-ui/core'
+import { PaymeRequestsTable } from './PaymeRequestsTable'
+import { PaymeLinksTable } from './PaymeLinksTable'
+
+const tabs = [
+  { label: 'Requests', key: 'requests', content: PaymeRequestsTable },
+  { label: 'Links', key: 'links', content: PaymeLinksTable },
+]
+
+export const PaymeSection = () => {
+  return (
+    <div className="space-y-2">
+      <div className="py-2">
```

Comment:
This div looks redundant to me. I think we can update the class of the parent div or the sibling heading to have the same space